lib/archethic/bootstrap/network_constraints.ex

```diff
@@ -0,0 +1,136 @@
+defmodule Archethic.Bootstrap.NetworkConstraints do
```

Comment:
I don't think this module is necessary.
It's even too complex.

Reply:
I tried those method, many like getting last txn , enforce loading of nss txs, this method evolved lot of experimentations, of retrieving. we can derive gen addr of nss and reward. until the node is authorized .because  in nss we require to have seed , which is warped secret and not accessible until authorised. I dont know how to proceed futher.

Reply:
Its logic is that keep trying to persist gen addr with node update msg, once persists exit normally, so transient.?

Reply:
You don't even need to access the NSS seed I think just access to the genesis address

Reply:
Resovles:Enforce resync of last network transactions after bootstrap#508 #515 .
After forced resync we can load gen addr from DB